packages/react-menu/e2e/Menu.e2e.ts

```diff
@@ -6,7 +6,7 @@ const menuSelector = '[role="menu"]';
 
 describe('MenuTrigger', () => {
   it('should open menu when clicked', () => {
-    cy.visitStory('Menu', 'TextOnly')
+    cy.visitStory('Menu-react-menu', 'TextOnly')
```

Comment:
This is an unpleasant consequence of story title change.

Reply:
😲😲😲

Can't we just leverage story `Id` property as previously so that the URL does not contain the suffix ?

Reply:
Yeah, that will be much better. I will just add an explicit id to the default export in the stories file.

Reply:
long term I'd stay away from that ID

- duplication
- if it changes it is a regretion -> tests should fail
- for test, it should be encapulsated in single constant probably to not have it all around the place :)

Reply:
I would love to stay away from the ID but we either need to:
* use the suffix in each component storybook
* remove the suffix from the suite storybook

However, URLs matching is only required for ease of testing a deployed storybook and local component storybook without extra logic. We can remove it even now if we prefer to add logic to our cypress test helpers instead.